Statut.md

```diff
@@ -0,0 +1,169 @@
+Rozdział I: Podstawowe informacje
+
+§1 Podstawowe informacje
+Podstawowe Fundacja pod nazwą Code For Poznań, zwana dalej „Fundacją”, ustanowiona przez Tomasza Magulskiego i Wojciecha Patelkę, zwanych dalej „Fundatorami”, aktem notarialnym sporządzonym w ... dnia ... roku, działa na podstawie ustawy z dnia 6 kwietnia 1984 r. o fundacjach (Dz. U. 1991, Nr 46, poz. 203, wraz z późniejszymi zmianami) oraz niniejszego Statutu.
+
+§2 Osobowość prawna
+Fundacja posiada osobowość prawną, którą nabywa z chwilą wpisania do rejestru fundacji w Krajowym Rejestrze Sądowym.
+
+§3 Siedziba Fundacji
+Siedzibą Fundacji jest miasto Poznań.
+
+§4 Teren działalności
+1. Fundacja prowadzi działalność na rzecz ogółu społeczności.
+2. Terenem działalności Fundacji jest obszar Rzeczpospolitej Polskiej, przy czym w zakresie niezbędnym dla właściwego realizowania celów może ona także prowadzić działalność, także poza granicami Rzeczypospolitej Polskiej.
+3. Fundacja może dla celów współpracy z zagranicą posługiwać się tłumaczeniem nazwy w wybranych językach.
+
+§5 Czas trwania Fundacji
+Czas trwania Fundacji jest nieograniczony.
+
+§6 Nadzór nad fundacją
+Ministrem sprawującym nadzór nad Fundacją jest minister właściwy do spraw gospodarki.
+
+§7 Identyfikacja Fundacji
+1. Pełną nazwą Fundacji jest Code For Poznań
+2. Fundacja może się posługiwać nazwą skróconą Code4 lub CodeFor oraz wyróżniającym ją znakiem graficznym.
+3. Fundacja używa pieczęci z oznakowaniem pełnej nazwy Fundacji oraz jej siedziby.
+
+Rozdział II: Cele i zasady działania Fundacji
+
+§8 Cel Fundacji
+Celem Fundacji jest udzielanie wsparcia w rozpowszechnianiu i wdrażaniu nowoczesnych rozwiązań technologicznych, pośród społeczności lokalnych i podmiotów, realizujących zadania społecznie użyteczne, poprzez wykorzystanie wiedzy programistycznej oraz tworzenie oprogramowania, które odpowiada na problemy społeczne, sprzyja aktywności lokalnej i promuje odpowiedzialność za wspólne otoczenie w praktyce społecznej oraz gospodarczej.
+
+§9 Realizacja celu Fundacji
```

Comment:
Liczba mnoga - celów?

Reply:
**(PUNKT DO DYSKUSJI)** Na kanwie rozmowy z Gosią doszliśmy do kilku punktów:
1) Po rozmowie z prawnikiem - istnieje różnica pomiędzy **stanem faktycznym**, a **działaniem**, które do niego zmierza. Cel organizacji powinien definiować "stan faktyczny osadzony w przyszłości", podczas gdy sposoby realizacji tego celu powinny opisywać sposoby i działania, które do niego zmierzają. Językowo więc, cel powinien definiować konkretny skutek, czy rezultat.
2) Cel powinien jakoś korespondować z ustawą o działalności pożytku publicznego i wolontariacie - art. 4.
3) Poszczególne projekty (np. Volontulo) mają bardziej szczegółowe cele (np. Art 4 ust. 27) _"promocja i organizacja wolontariatu"_) niż organizacja jako całość (np. Art 4 ust. 33) _"działanie na rzecz organizacji pozarządowych"_)
4) Po przejrzeniu całego katalogu w ustawie znaleźliśmy trzy cele, odpowiadające **całej organizacji**:
a) działalności na rzecz **organizacji pozarządowych** oraz podmiotów wymienionych w art. 3 ust. 3, w zakresie określonym w pkt 1–32a.
b) działalności wspomagającej rozwój wspólnot i **społeczności lokalnych**
c) działalności wspomagającej rozwój techniki, wynalazczości i innowacyjności oraz rozpowszechnianie i wdrażanie **nowych rozwiązań technicznych** w praktyce gospodarczej
_(Boldem zaznaczyłem frazy, które można by wykorzystać w formułowaniu celu, tak żeby korespondował z ustawą, ale też dobrze/trafnie opisywał działalność Code for Poznań)_
5) Cel fundacji rzecz jasna pełni funkcję formalną, ale też odpowiada na pytanie "kim jesteśmy?". Dlatego myślę, że warto się zmęczyć i zmóżdżyć to zdanie, tak żeby wryło się w pamięć, jednocześnie jasno tłumacząc, co jest pięć. "Tworzymy technologiczne rozwiązania problemów społecznych" jest super, ale wymaga jeszcze pokazania Volontulo i dopowiedzenia dwóch kolejnych zdań :D

Reply:
A no i 6. myślę, że @w1stler @magul może jeszcze z Przemkiem - to Wy powinniście wykrztusić to zdanie :)